### PR TITLE
Refresh install docs

### PR DESCRIPTION
**Pull request by taylorsilva**
See commit messages for details.

### CODE REVIEW

File: lit/docs/install/concourse-cli.lit [hunk 1]
```diff
@@ -9,11 +9,20 @@ contains the following files:
...
-concourse/resource-types/... # on Linux
+concourse/resource-types/... # Linux only
 }}
 
+The Linux release is the largest among all the platforms because it contains
+some of the most commonly used resource types like the
+\link{git}{https://github.com/concourse/git-resource},
+\link{time}{https://github.com/concourse/time-resource/}, and
+\link{registry-image}{https://github.com/concourse/registry-image-resource/}
+resources. Resources only run on Linux workers, that's why the other platforms
```
Comment (chenbh) [inline lit/docs/install/concourse-cli.lit:22]: We could also point to the "Bundled resource types" section of any release newer than v6.7.3

File: lit/docs/install/concourse-cli.lit [hunk 2]
```diff
@@ -9,11 +9,20 @@ contains the following files:
...
+are not bundled with resources; resources don't currently exist for non-linux
+platforms.
+
 When extracted, the \code{concourse} binary will auto-discover its sibling
```
Comment (chenbh) [inline lit/docs/install/concourse-cli.lit:26]: Can you also detail where these resources are located and how to figure out what resource versions are deployed on each worker? 

- `fly workers --details` which gives resource types but no versions
- `fly curl api/v1/workers` which gives bulk response for resource types + versions for every worker
- hopping onto a worker and reading the `metadata.json`

File: lit/docs/install/worker.lit [hunk 3]
```diff
@@ -76,53 +77,51 @@ decide much on its own.
...
-    }{
-      repeated connections to \code{7777} and \code{7788} from the
-      \reference{web-node}'s TSA component as it heartbeats to ensure the worker
-      is healthy
+      From the \reference{web-node} on port \code{7777} (Garden) and
+      \code{7788} (BaggageClaim). These ports do not need to be exposed, they
+      are forwarded to the web node via the ssh connection on port 2222.
     }
   }
 }
```
Comment (chenbh) [inline lit/docs/install/worker.lit:127]: There's inbound + outbound traffic to other worker nodes if p2p streaming is enabled

File: lit/docs/install/worker.lit [hunk 4]
```diff
@@ -211,7 +214,7 @@ decide much on its own.
     When a worker machine is going away, it should be \italic{retired}. This is
     similar to \italic{landing}, except at the end the worker is completely
     unregistered, along with its volumes and containers. This should be done
-    when a worker's VM or container is being destroyed.
+    when a worker's VM or container are being destroyed.
```
Comment (chenbh) [inline lit/docs/install/worker.lit:217]: I think the `is` is appropriate here - it's referring to the worker VM/container itself which is singular

File: lit/docs/install/concourse-cli.lit (hunk 2 above)
Comment (taylorsilva) [reply lit/docs/install/concourse-cli.lit:26]: Great idea. I'll do this in the worker section.

**Comment by taylorsilva** [comment]
@chenbh Feedback addressed. Add two new sections to the workers page about resource types and tagged/team workers.

File: lit/docs/install/worker.lit [hunk 5]
```diff
@@ -239,41 +246,225 @@ decide much on its own.
...
+    \reference{managing-teams}{team} then you can configure a worker to belong
+    to a single team like so:
+
+    \codeblock{bash}{{
+    CONCOURSE_TEAM="lightweavers"
+    }}
+
+    Once an untagged team worker is registered Concourse will schedule all
+    untagged builds for that team on its team worker(s). Builds for this team
+    will not be scheduled on any untagged, non-team workers anymore.
```
Comment (chenbh) [inline lit/docs/install/worker.lit:283]:
```suggestion
    Once an untagged team worker is registered Concourse will schedule all
    untagged builds for that team on its team worker(s). Builds for this team
    will no longer be scheduled on any untagged, non-team workers.
```

File: lit/docs/install/worker.lit [hunk 6]
```diff
@@ -239,41 +246,225 @@ decide much on its own.
...
+
+    It is possible to have a Concourse cluster made up of only team workers and
+    have zero non-team workers, though this is not a common setup because
+    resource uitilization across all workers ends up underuitilized. It is
+    useful though if you have a particular team with heavy worloads that
+    usually bothers other teams pipelines.
```
Comment (chenbh) [inline lit/docs/install/worker.lit:289]:
```suggestion
    resource utilization across all workers ends up under-utilized. It is
    useful though if you have a particular team with heavy workloads that
    usually bothers other teams pipelines.
```

File: lit/docs/install/worker.lit [hunk 7]
```diff
@@ -239,41 +246,225 @@ decide much on its own.
...
+      We will use the \link{git
+      resource}{https://github.com/concourse/git-resource} as our example. We
+      will assume your Concourse installation is at \code{/usr/local/concourse}.
+
+      First, pull and create a container of the resource you're
+      installing/upgrading. The container does not need to remain running.
+
+      \codeblock{bash}{{
+      docker pull concourse/git-resource
+      docker run concourse/git-resource
```
Comment (chenbh) [inline lit/docs/install/worker.lit:360]: running in detached mode returns the container id immediately
```bash
$ docker run -d concourse/git-resource
24e8123428e8317f63bb94bd91e589e8ba62f0ed80cc25564820169f6f5ce367
```

File: lit/docs/install/worker.lit [hunk 8]
```diff
@@ -239,41 +246,225 @@ decide much on its own.
...
+      }}
+
+      Export the containers file system into a tar archive. Then gzip compress
+      the tar archive into a file named \code{rootfs.tgz}
+
+      \codeblock{bash}{{
+      docker export a895ae2dbb24 -o rootfs.tar
+      gzip -c rootfs.tar > rootfs.tgz
+      rm rootfs.tar
+      }}
```
Comment (chenbh) [inline lit/docs/install/worker.lit:378]: both docker and gzip works with stdin/out by default

```suggestion
      \codeblock{bash}{{
      docker export a895ae2dbb24 | gzip > rootfs.tgz
      }}
```

File: lit/docs/install/worker.lit [hunk 9]
```diff
@@ -303,9 +494,12 @@ decide much on its own.
       }
     }
     \section{
-      \title{\bold{Transitioning from Guardian to containerd}}
+      \title{{Transitioning from Guardian to containerd}}
```
Comment (chenbh) [inline lit/docs/install/worker.lit:497]:
```suggestion
      \title{Transitioning from Guardian to containerd}
```

File: lit/docs/install/worker.lit (hunk 7 above)
Comment (taylorsilva) [reply lit/docs/install/worker.lit:360]: niceeeeeeee

File: lit/docs/install/worker.lit (hunk 8 above)
Comment (taylorsilva) [reply lit/docs/install/worker.lit:378]: didn't know that about gzip! Docker has a funny message when you try to export to a terminal haha

File: lit/docs/install/worker.lit (hunk 6 above)
Comment (taylorsilva) [reply lit/docs/install/worker.lit:289]: Can you tell I forgot to `:set spell` here 😅

**Review by chenbh — APPROVED** [review]
lgtm

**Pull request merged**